src/main/java/com/solace/samples/jcsmp/snippets/HowToConfigureSessionUsingOAUTH2.java

```diff
@@ -16,55 +16,106 @@
 
 package com.solace.samples.jcsmp.snippets;
 
-import com.solacesystems.jcsmp.JCSMPProperties;
+import com.solacesystems.jcsmp.*;

```

Comment:
did best practices change? shouldn't we be listing out the individual packages?

Reply:
Multiple classes from the same package impacts readability. Hence IntelliJ code completion compresses them to a single * based import